collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,382 @@
+
+![A compute texture demo from the official collection of Godot demos](/storage/blog/godot-4-2-beta/compute-shader-textures.webp)
+
+If you are using compute shaders, a pair of goodies should make your life way more exciting. First is the newly added ability to create custom texture objects ([GH-79288](https://github.com/godotengine/godot/pull/79288)). You can see how it works in practice to create water effects with this [soon-to-be merged compute texture demo](https://github.com/godotengine/godot-demo-projects/pull/938). And another submission adds a set of APIs to call compute code on the render thread, for cases when you need to synchronize compute shaders and rendering ([GH-79696](https://github.com/godotengine/godot/pull/79696)).
+
+A prominent bug has been fixed, which caused a crash if you went over 204 lights, decals, and reflection probes, combined ([GH-80845](https://github.com/godotengine/godot/pull/80845)). This issue stopped users from fully flexing the strength of the Forward+ renderer, but now they are free to use up to 512 of each entity type in the default configuration (this limit can be increased). And the built-in glow effect has been improved, making its visuals closer to the high quality look from Godot 3.x ([GH-82353](https://github.com/godotengine/godot/pull/82353) by [Raffaele Picca](https://github.com/RPicster)).
+
+And now, for the big hitters...
+
+#### New features
+
+Godot 4.2 features support for AMD's <abbr title="FidelityFX Super Resolution">FSR</abbr> 2.2, an open upscaling technique that works on GPUs of all vendors ([GH-81197](https://github.com/godotengine/godot/pull/81197)). This is possible thanks to [Darío Samo](https://github.com/DarioSamo), who also implemented a prerequisite support for motion vectors in skeletons, blend shapes, and particles ([GH-80618](https://github.com/godotengine/godot/pull/80618), [GH-80688](https://github.com/godotengine/godot/pull/80688)), and also solved many issues with the current <abbr title="Temporal anti-aliasing">TAA</abbr> system that have been plaguing users.
+
+Darío also implemented a new lightmapper denoising approach to replace the old, extremely bulky and slow <abbr title="OpenImage Denoise">OIDN</abbr> denoiser. Although a newer and improved version of OIDN is available, we were unfortunately stuck with an older one due to technical incompatibilities. The new <abbr title="Joint Non-Local Means">JNLM</abbr> denoiser avoids similar issues. It is also much faster as it utilizes compute shaders (and thus your GPU), and it's significantly more lightweight ([GH-81659](https://github.com/godotengine/godot/pull/81659)). While the quality decrease can be noticeable under some circumstances, thanks to drastically reduced baking times, you can now use higher resolution maps to counteract that.
```

Comment:
Since https://github.com/godotengine/godot/pull/82832 was merged, it probably should be mentioned that new versions of OIDN are usable as external tool.